snapshot-labs/snapshot.js
# feat: Add limits for voting delay and period

## Changes:
- Refer https://discord.com/channels/955773041898573854/1030772407226601522/1210204913381023794
- Add one month limit to voting delay
- Add 6 month limit for voting period 
- Add custom errors using `errorMessage` so error will be `"Delay must be less than 30 days"` instead of `Must be < 2592000`


## How to test:
- Run `yarn test` 
- all tests should pass
- Change values in `test/examples/space.json` to something more than limits
- Notice the new error message
